src/Provider/Session/Session.php

```diff
@@ -9,7 +9,9 @@ class Session implements SessionInterface
 {
     public function __construct()
     {
-        if (session_status() == PHP_SESSION_NONE) {
+        if (session_status() === PHP_SESSION_NONE &&
+            !(PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg')
```

Comment:
Would it perhaps make sense to be more open to future additions to make that an ```in_array```-call?

Like this:

```php
$noSession = [
    'cli', 
    'phpdbg',
];

if (session_status() === PHP_SESSION_NONE && 
    ! in_array(PHP_SAPI, $noSession))
{
```

Reply:
Perhaps the `$noSession`-Variable could also be a static class-variable…

Reply:
I don't foresee any other SAPI for which this might be required. We can always change in future when the need arises :)